sdk/communication/Azure.Communication.Common/api/Azure.Communication.Common.netstandard2.0.cs

```diff
@@ -9,6 +9,14 @@ public abstract partial class CommunicationIdentifier
     {
         protected CommunicationIdentifier() { }
     }
+    public sealed partial class CommunicationTokenCredential : System.IDisposable
+    {
+        public CommunicationTokenCredential(Azure.Communication.Identity.CommunicationTokenRefreshOptions tokenRefreshOptions) { }
+        public CommunicationTokenCredential(string userToken) { }
```

Comment:
Let's rename the param to `token` and don't mention user

Reply:
Done